data/data/bootstrap/files/etc/systemd/system-generators/node-image-overlay-generator

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+
+UNIT_DIR="${1:-/tmp}"
+
+if ! rpm -q openshift-clients &>/dev/null; then
```

Comment:
@eranco74 @carbonin this is the proxy test for whether the current filesystem needs all of the "node layer" stuff to be layered on. Any concerns for the assisted-installer live ISO? We don't install this, right?

Reply:
No, we don't install anything additional on the rootfs